doc/node.1

```diff
@@ -1,28 +1,54 @@
 .TH NODE 1 2016 Node.js Node.js
 
-.\ This is a man page comment.
-.\ Man page syntax (actually troff syntax) is somewhat obscure, but the
-.\ important part is is that .<letter> specifies <letter>'s syntax for that
-.\ line, and \f<letter> specifies it for the characters that follow.
-.\ See http://liw.fi/manpages/ for more info.
+.\" This is a man page comment.
+.
+.\" Man page syntax (actually troff syntax) is somewhat obscure, but the
+.\" important part is is that .<letter> specifies <letter>'s syntax for that
+.\" line, and \f<letter> specifies it for the characters that follow.
+.
+.\" .B   Bold line
+.\" .I   Italic line (Rendered as underlined text in terminals)
+.\" .BI  Alternating bold/italics without spaces between arguments.
+.\"      Use `\ ` to include an "unpaddable" (literal) space in the output.
+.\" .RI  Alternating roman/italic
+.
+.\" See http://liw.fi/manpages/ for an overview, or http://www.troff.org/54.pdf
+.\" for detailed language reference.
+.de ur
+.nr CF \\n(.f
+.ft 4
+\\$1
+.ft \\n(CF
+..
+
 
 .SH NAME
 
 node \- Server-side JavaScript runtime
 
 
 .SH SYNOPSIS
-
+.fi
+.
```

Comment:
`.fi`: That turns on filling mode, which I forgot is enabled by default. Turning it off with `.nf` would result in this:
<img width="318" alt="figure-1" src="https://cloud.githubusercontent.com/assets/2346707/17029072/1300eb8a-4fad-11e6-85b8-8c4782da5b55.png" />

`.`: That's the safe way of inserting blank lines for readability in a roff document. Inserting a blank line-break without a control-character tells the interpreter that the line is to be interpreted as content, and it adds another line-break onto the output (which is moot if filling-mode is enabled anyway).